tests/Basic/Basic.h

```diff
@@ -760,15 +760,10 @@ class TestNamingAnonymousTypesInUnion
 };
 
 #define CS_API
-class CS_API ClassPassTry
-{
-public:
-	int n;
-	char c;
-};
+class CS_API RefTypeClassPassTry { };
```

Comment:
When I wrote this test the first time then it was to address an issue which
specifically mentioned failure on using CS_API, so I left it as it was.
Should I change it to DLL_API?!
On 28-Aug-2015 5:52 PM, "João Matos" <notifications@github.com> wrote:

In tests/Basic/Basic.h
<https://github.com/mono/CppSharp/pull/548#discussion_r38194070>:

> @@ -760,15 +760,10 @@ class TestNamingAnonymousTypesInUnion
>  };
>
>  #define CS_API
> -class CS_API ClassPassTry
> -{
> -public:
> -	int n;
> -	char c;
> -};
> +class CS_API RefTypeClassPassTry { };

What's the purpose of adding CS_API here and not DLL_API?

—
Reply to this email directly or view it on GitHub
<https://github.com/mono/CppSharp/pull/548/files#r38194070>.